src/pages/VeSDL/LockedInfo.tsx

```diff
@@ -60,10 +60,10 @@ export default function LockedInfo(): JSX.Element {
           {aggSDLInfo ? formatBNToShortString(aggSDLInfo.totalVeSDL, 18) : "-"}
         </Typography>
       </Box>
-      <Box flex={1}>
+      {/* <Box flex={1}>
```

Comment:
Plans with this?

Reply:
https://github.com/saddle-finance/saddle-frontend/issues/1092